src/components/Card/components/CardFooter/CardFooter.tsx

```diff
@@ -1,34 +1,60 @@
 import React, { FC, ReactNode } from 'react';
 import classNames from 'classnames';
 import { Box, BoxProps } from '../../../Box/Box';
+import { BrandColor } from '../../../../types';
 import styles from '../../Card.module.scss';
 
 export interface CardFooterProps extends BoxProps {
   /**
    * Contents of the Footer.
    */
   children?: ReactNode;
+  /**
+   * If defined as a prop, this value will take higher precedence than the corresponding component design token value
+   * Any valid [brand color token](/?path=/story/design-tokens-design-tokens--page#color), or a `url()` for an image
+   */
+  background?: BrandColor;
```

Comment:
To ensure consistency across these props in the Box and the CardFooter, I would change these to pull directly from BoxProps. E.G:

`background?: BoxProps['background'];`